### PR TITLE
Render canvas subcards with delay

### PR DESCRIPTION
**Pull request by Serhiy91**

### CODE REVIEW

**Comment by Dartv** [comment]
I found the easiest solution without changing the core library functionality. You just need to move the declaration inside component render function. That's it!

**Comment by Dartv** [comment]
Also, delete libreact because we only use `delayed` from there

**Comment by Dartv** [comment]
Because of that, we can make an exception and create wrapper component `DelayedCanvasSubcards` and reuse it everywhere.

File: app/imports/client/react/canvas/components/CustomerSegmentEditModal.js [hunk 1]
```diff
@@ -73,155 +66,162 @@ const CustomerSegmentEditModal = ({
...
-              {...{ initialValues }}
-              validate={validateCustomerSegment}
-              onSubmit={(values, form) => {
-                const currentValues = getInitialValues(data);
-                const difference = diff(values, currentValues);
+}) => {
+  const DelayedCustomerSegmentSubcards = delayed({
+    loader: () => import('./CustomerSegmentSubcards'),
+    idle: true,
+    delay: 250,
```
Comment (Dartv) [inline app/imports/client/react/canvas/components/CustomerSegmentEditModal.js]: How much do you save in initial bundle to justify using dynamic import? And if you use dynamic import, there's no need to use delay

Comment (Serhiy91) [reply app/imports/client/react/canvas/components/CustomerSegmentEditModal.js]: 2kB

File: app/imports/client/react/canvas/components/DelayedCanvasSubcards.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import { delayed } from '../../helpers';
+import CanvasSubcards from './CanvasSubcards';
 
 const DelayedCanvasSubcards = props => delayed({
-  loader: () => import('./CanvasSubcards'),
+  loader: () => CanvasSubcards,
```
Comment (Dartv) [inline app/imports/client/react/canvas/components/DelayedCanvasSubcards.js]: You need to return a promise, don't you?

Comment (Serhiy91) [reply app/imports/client/react/canvas/components/DelayedCanvasSubcards.js]: @Dartv Added

**Comment by Dartv** [comment]
Can you delete named import in helpers and instead import `delayed `directly? This enhancer is only needed in canvas screen and we don't want to load it elsewhere

**Comment by Serhiy91** [comment]
@Dartv Done

**Comment by Dartv** [comment]
@Serhiy91 I had to revert this PR because when you edit any subcard all of subcards disappear/reappear

**Pull request merged**